projects/uitdatabank/docs/entry-api/organizers/create.md

```diff
@@ -1,9 +1,161 @@
-Coming soon.
-
-<!--
-  @todo
-  - Mention that you must always reuse an existing organizer when possible and link to that guide.
-  - Focus on creating a simple organizer with just the required properties here.
-  - Explain that you can also supply optional properties and give examples
-  - Link to some other guides as next steps (for example "Updating an organizer", ...)
--->
+# Creating a new organizer
+
+This guide explains how to create a new organizer in UiTdatabank using Entry API. You'll learn about all the required properties of an organizer and how to define them, and see examples of various possible organizers.
+
+<!-- theme: warning -->
+
+> Creating duplicate organizers is not allowed, you must reuse existing ones. See [finding and reusing organizers](./finding-and-reusing-organizers.md)
+
+Before diving in, make sure you have read the following guides first:
+
+* [Authentication](../authentication.md)
+* [What are organizers?](introduction.md)
+
```

Comment:
```suggestion
<!-- theme: warning -->

> When sending event content to UiTdatabank through the Entry API you **must** reuse existing organizer IDs when available. The creation of a new organizer is only allowed if the organizer does not exist yet in UiTdatabank. Otherwise, your integration can be blocked because of the creation of duplicates.
```